helios/stats_views.py

```diff
@@ -40,13 +40,15 @@ def elections(request):
 
   page = int(request.GET.get('page', 1))
   limit = int(request.GET.get('limit', 25))
+  q = request.GET.get('q','')
```

Comment:
By default in Django template, every variable is automatically escapes. You can see this at https://docs.djangoproject.com/en/1.11/ref/templates/language/#automatic-html-escaping. 
I think this is enough for XSS protection. What do you think about this?